### ISSUE TITLE
Please add support for S40ZBTPB Lite 

### ISSUE THREAD

**Issue by trhessel** [bwp91/homebridge-ewelink#532]
Please add support for the following:

12/16/2023, 12:28:46 PMeWeLink BridgeeWeLinkWARNING[Device022f18] is not supported, do create a GitHub issue with the following info:
[7005] {"bindInfos":{},"subDevId":"f358e829004b12007005","parentid":"1001dc6ebb","fwVersion":"1.1.0","switch":"off"}.

Thank you

**Comment by bwp91** [comment]
Hi @trhessel 

Do you still have this message on v12.1.0 of the plugin?

**Comment by trhessel** [comment]
No, all good and I am able to control this via homekit. Thank you for
adding this and following up.
